src/navigation/context.py

```diff
@@ -227,6 +242,9 @@ def __init__(self):
         self.tf_listener = tf2_ros.TransformListener(self.tf_buffer)
         self.vel_cmd_publisher = rospy.Publisher("cmd_vel", Twist, queue_size=1)
         self.vis_publisher = rospy.Publisher("nav_vis", Marker, queue_size=1)
+        self.search_point_publisher = rospy.Publisher("search_path", GPSPointList, queue_size=1)
+        self.gate_path_publisher = rospy.Publisher("gate_path", GPSPointList, queue_size=1)
```

Comment:
Based on the ICD and the PR from Teleop, I thought this should be /projected_points instead of /search_path and /gate_path. Is this not up to date or is something else amiss?

Tagging @emilyyysu @yshh00 for awareness

Reply:
ah yea I talked to joe about this sorry I've updated the ICD and will check in on that PR as well.